backend/persistence/models/audit_log.go

```diff
@@ -0,0 +1,46 @@
+package models
+
+import (
+	"github.com/gofrs/uuid"
+	"time"
+)
+
+type AuditLog struct {
+	ID                uuid.UUID    `db:"id" json:"id"`
+	Type              AuditLogType `db:"type" json:"type"`
+	Error             string       `db:"error" json:"error"`
+	MetaHttpRequestId string       `db:"meta_http_request_id" json:"meta_http_request_id"`
+	MetaSourceIp      string       `db:"meta_source_ip" json:"meta_source_ip"`
+	MetaUserAgent     string       `db:"meta_user_agent" json:"meta_user_agent"`
+	ActorUserId       *uuid.UUID   `db:"actor_user_id" json:"actor_user_id"`
+	ActorEmail        string       `db:"actor_email" json:"actor_email"`
```

Comment:
Why do we need the Actor email in this table? Isn't this redundant?

Reply:
Well you think it is redundant because of the `ActorUserId`? In that case, no it is not redundant, because maybe the user was deleted  or the user changed its email and then you don't have the email, which means you might loose important information.

Reply:
Ahh alright, true. So it captures the identifier at the time the action did take place. Makes sense.